src/sched.rs

```diff
@@ -35,6 +35,39 @@ bitflags!{
     }
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct CloneFlagsArg {
+    flags: CloneFlags,
+    signal: Option<c_int>,
+}
+
+impl CloneFlagsArg {
+    fn bits(self) -> c_int {
+        self.flags.bits() | self.signal.unwrap_or(0)
```

Comment:
Yes it does. Because only the low 8 signals are allowed if I understand the manpage correctly.